osu.Game/Overlays/Comments/DrawableComment.cs

```diff
@@ -0,0 +1,356 @@
+                                                        Font = OsuFont.GetFont(size: 12),
+                                                        Text = HumanizerUtils.Humanize(comment.CreatedAt)
+                                                    },
+                                                    new RepliesButton(comment.RepliesCount)
+                                                    {
+                                                        Expanded = { BindTarget = childrenExpanded }
+                                                    },
+                                                }
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    childCommentsVisibilityContainer = new FillFlowContainer
+                    {
+                        RelativeSizeAxes = Axes.X,
+                        AutoSizeAxes = Axes.Y,
+                        Direction = FillDirection.Vertical,
+                        Children = new Drawable[]
+                        {
+                            childCommentsContainer = new FillFlowContainer
+                            {
+                                Padding = new MarginPadding { Left = 20 },
+                                RelativeSizeAxes = Axes.X,
+                                AutoSizeAxes = Axes.Y,
+                                Direction = FillDirection.Vertical
+                            },
+                            deletedChildrenPlaceholder = new DeletedChildrenPlaceholder
+                            {
+                                ShowDeleted = { BindTarget = ShowDeleted }
+                            }
+                        }
+                    }
+                }
+            };
+
+            deletedChildrenPlaceholder.DeletedCount.Value = comment.GetDeletedChildrenCount;
+
+            if (comment.UserId.HasValue)
+                username.AddUserLink(comment.User);
+            else
+                username.AddText(comment.LegacyName);
+
+            if (comment.EditedAt.HasValue)
+            {
+                info.Add(new SpriteText
+                {
+                    Anchor = Anchor.CentreLeft,
+                    Origin = Anchor.CentreLeft,
+                    Font = OsuFont.GetFont(size: 12),
+                    Text = $@"edited {HumanizerUtils.Humanize(comment.EditedAt.Value)} by {comment.EditedUser.Username}"
+                });
+            }
+
+            if (!comment.IsDeleted)
+            {
+                var formattedSource = MessageFormatter.FormatText(comment.GetMessage());
+                message.AddLinks(formattedSource.Text, formattedSource.Links);
+            }
+            else
+            {
+                content.FadeColour(OsuColour.Gray(0.5f));
```

Comment:
`content` is effectively empty yeah? It needs to contain the message for admins when `ShowDeleted = true`.

Reply:
![image](https://user-images.githubusercontent.com/1329837/66815116-ad0aff00-ef72-11e9-812b-6701b4cf9e44.png)

Reply:
So API sends the `MessageHtml` of deleted comments only for several types of users, right?

Reply:
For admins, yup.

Reply:
Should be ok now